### ISSUE TITLE
user interface: QC: Report any occurrences where the allele descriptions disagree with a warning

### ISSUE THREAD

**Issue by pombase-admin** [pombase/canto#630]
(Transferred from[curation-tool:#451])

If the descriptions disagree, tell the user:
"This allele has previously been described as xxxx, please contact curators if your 
 allele has a different description"

Exception 1: if one of the descriptions is "unknown" or "other" in which case an informative 
description can replace. 
example   for case (from cdc2) cdc2-3w (unknown) and cdc2-3W (C67Y)

Exception 2: We might sometimes describe an allele as its amino acid sequence and sometimes 
the  nucleotide sequence (curators  have decided that if we have both available we will capture 
nucleotide because it is more informative.   However, in different papers we may be provided only with the AA change, so it is possible we will  capture both in which case we would like to store both. 

It should be possible to figure out if the nucleotide and protein change reported are equivalent  (for example A2F (amino acid) = x4|5|6y nucleotide and only report a warning if they disagree.
(in PomBase we will show amino acid change, but the nucleotide will show when you mouse over…
add to jira tracker)

Original comment by: kimrutherford

**Comment by ValWood** [comment]
This isn't super urgent, it needs doing eventually. 
It has a "quick" label, but I suspect that, as it has been on the tracker since 2013, that this is not true?

So, for now it can be "low priority" its definitely lower than lots of stuff...

**Comment by kimrutherford** [comment]
> If the descriptions disagree, tell the user:

Do you mean if the name and type are the same, but the descriptions disagree?

**Comment by ValWood** [comment]
I don't know how much checking is already done.

People need a warning if  the gene,name and type are the same and the descriptions disagree
(it is possible that people have used the same allele name for different alleles of the same gene, but I would expect this to be rare.

We also need to 'merge' any alleles with the name descriptions, but different names. Then we would need to decide which was the allele name and which was the synonym.

We also need to recognise that we might have alleles described as their nucleotide, or protein
mutational effect, and that both are valid, and what to do in these cases. My feeling is that the 
protein mutation is preferred, However there is slightly more information in the nucleotide description  (you can infer protein from nucleotide, but not nucleotide from protein).

So, there is quite a lot to think about here.
Could you summarize what happens right now?
We can then discuss what should happen at our upcoming curator meeting (we will have a log file/allele sorting jamboree so this topic will fit quite well).

**Comment by ValWood** [comment]
I think this can close (was looking for something else).
We have log checks  for this and it isn't really an issue.
